src/components/Header/Header.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react'
+import { NavLink, Link } from 'react-router-dom'
+import { Menu, MenuItem, MenuMenu, Icon, Label, Image, Popup, PopupContent, 
+    ItemGroup, Item, ItemImage, ItemContent, ItemHeader, ItemMeta } from 'semantic-ui-react'
```

Comment:
Is this all on the same line ? Seems weird, maybe trying to align the parameters